doc/nrf/config_and_build/bootloaders_and_dfu/bootloader_adding.rst

```diff
@@ -59,10 +59,18 @@ To ensure that the immutable bootloader occupies as little flash memory as possi
 
 See :ref:`ug_bootloader_config` for more information about using Kconfig fragments.
 
+.. caution::
+   If you want to perform DFU, remember to :ref:`define a static partition map <ug_pm_static>` for the immutable bootloader.
```

Comment:
Maybe we can keep the note right under `Adding a bootloader chain`?
It's a common requirement no matter which bootloader is going to be used.